src/data-tutorials/users.tsx

```diff
@@ -912,6 +912,13 @@ const Users: User[] = [
     website: 'blog/tutorials/flutter-celo-easily-build-flutter-mobile-dApps',
     tags: ['intermediate','foundation', 'flutter', 'composer'],
   },
+  {
+    title: 'FloralNft Smart Contract for Buying and Gifting Flowers as NFTs',
+    description: 'FloralNft is a smart contract built on the Celo blockchain that allows users to create and trade unique digital flowers as non-fungible tokens (NFTs).',
+    preview: require('./showcase/intermediate/FloralNft-Smart-Contract-for-Buying and-Gifting-Flowers as-NFTs.png'),
```

Comment:
Missing - in some part -./showcase/intermediate/FloralNft-Smart-Contract-for-Buying-and-Gifting-Flowers-as-NFTs.png